### ISSUE TITLE
Product Design

### ISSUE THREAD

**Issue by Hornet004** [playproject-io/contracting#15]
## `@todo`
- [ ] Product Design
    - `@input` ❓ `solution concept` from #14  
    - `@input` ❓ `concept Validation` from #14  

    * [ ] process the inputs to generate the outputs
    - `@output` ❓ `wireframes/Iterations`
      - next #16 
    - `@output` ❓ `ui designs`
      - next #16 
---

### Summary
In this directory, we use the information provided from #14 to structure wireframes and create the actual design.
### `wireframes/Iterations`
- Prepare wireframes using Figma link for validations from @serapath 
### `wireframes validations`
- @serapath  verifies wireframes and give feedbacks. 
### `UI designs`
- After wireframe validations, `wireframe` is converted into actual Figma UI as `input`
- Figma file is presented after, for `web development`

**Comment by serapath** [comment]
**feedback `2022.04.27`**

Sweet, so here the same feedback as for issue #14 applies, but ... as you say above:

> In this directory, we use the information provided from https://github.com/playproject-io/contracting/issues/14 to structure wireframes and create the actual design.

**this is great** :+1: :100: 

Now the better way to represent that in the issue structure would be, because you say **`"we use the information provided by"`**, to edit this issue above and instead write it like this:

```
## `@todo`
- [ ] Product Design
    - `@input` ❓ `solution concept` from #14
    - `@input` ❓ `concept Validation` from 14
    * [ ] process the inputs to generate the outputs
    - `@output` ❓ `wireframes/Iterations`
    - `@output` ❓ `ui designs`

```

...now maybe `process the inputs to generate the outputs` is a bit generic and dry, so you could instead describe what your exact plans are :-)

...oh you would also edit the previous issue #14 to change it like so:

```
## `@todo`
- [ ] Information Architecture
    - `@output` ❓ `solution concept`
      - next #15
    - `@output` ❓ `concept Validation`
      - next #15

---

## `@info`
...

```
 
so you would know already in issue #14 that those planned outputs will be used as inputs in #15 :-)

**Comment by serapath** [comment]
merged into #49
